extensions/blocks/anchor-fm/editor.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import { createBlock } from '@wordpress/blocks';
+import { dispatch } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { name } from '.';
+import getJetpackExtensionAvailability from '../../shared/get-jetpack-extension-availability';
+import { waitForEditor } from '../../shared/wait-for-editor';
+
+async function insertSpotifyBadge() {
+	const { spotifyShowUrl } = window.Jetpack_AnchorFm;
+	if ( ! spotifyShowUrl ) {
+		return;
+	}
+
+	await waitForEditor();
+
+	dispatch( 'core/block-editor' ).insertBlock(
+		createBlock( 'core/image', {
+			url: 'https://cldup.com/Dv6JZWyRpq-1200x1200.png',
+			linkDestination: 'none',
+			href: spotifyShowUrl,
+			align: 'center',
+			width: 165,
+			height: 40,
+			className: 'is-spotify-podcast-badge',
+		} ),
+		0,
+		undefined,
+		false
+	);
+}
+
+function initAnchor() {
+	const isExtensionAvailable = getJetpackExtensionAvailability( name )?.available;
```

Comment:
if the block [doesn't require a paid plan](https://github.com/Automattic/jetpack/pull/17936/files#r537721223), maybe checking its availability should be removed.

Reply:
I think we still need to check it to deal with the availability of beta extensions?

Reply:
Oh, good call. Not sure honestly.

Reply:
I've tested the patch on dotcom, and I think it isn't necessary. [Beta extension section](https://github.com/Automattic/jetpack/tree/master/extensions/#beta-extensions) in the doc page doesn't mention its usage, either.